flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/operators/window/groupwindow/operator/WindowOperatorContractTest.java

```diff
@@ -243,7 +244,8 @@ KeyedOneInputStreamOperatorTestHarness<RowData, RowData, RowData> createWindowOp
                             sendRetraction,
                             allowedLateness,
                             UTC_ZONE_ID,
-                            -1);
+                            -1,
+                            0);
```

Comment:
could we add tests to check that a non zero stateRetentionTime works as well please.